data-serving/data-service/test/model/travel.test.ts

```diff
@@ -0,0 +1,31 @@
+import { TravelDocument, travelSchema } from '../../src/model/travel';
```

Comment:
Planning to json this, like the other ones?

Reply:
sure -- I'll do it here